erniebot-agent/erniebot_agent/tools/base.py

```diff
@@ -294,20 +308,23 @@ async def __call__(self, **tool_arguments: Dict[str, Any]) -> Any:
             )
 
         # parse the file from response
-        returns_file_names = get_file_names_from_param_view(self.tool_view.returns)
-        file = await parse_file_from_response(response, self.file_manager, file_names=returns_file_names)
+        returns_file_mimetypes = get_file_mimetype_from_param_view(self.tool_view.returns)
+        file = await parse_file_from_response(
+            response, self.file_manager, file_mimetypes=returns_file_mimetypes
+        )
 
         if file is not None:
-            if len(returns_file_names) == 0:
+            if len(returns_file_mimetypes) == 0:
                 return {self.tool_view.returns_ref_uri: file.id}
 
-            return {returns_file_names[0]: file.id}
+            file_name = list(returns_file_mimetypes.keys())[0]
+            return {file_name: file.id}
 
-        if len(returns_file_names) == 0:
+        if len(returns_file_mimetypes) == 0:
             return response.json()
 
         raise RemoteToolError(
-            f"You have defined the files: <{returns_file_names}>, but can not parse file from response. "
+            f"You have defined the files: <{returns_file_mimetypes}>, but can not parse file from response. "
             "Please make sure that there are `Content-Disposition` or `Content-Type` field "
```

Comment:
<{returns_file_mimetypes}> are defined but cannot be processed from the response. Please ensure that the response headers contain either the `Content-Disposition` or `Content-Type` field.